### PR TITLE
Enable sim hopper to dispense over time

### PR DESCRIPTION
**Pull request by rsinnet**
This PR refactors the sim hopper and adds a `LinearDispenser` which dispenses food into the scale over time.

### CODE REVIEW

File: miso_peripherals/src/miso_peripherals/hopper/sim_driver.py [hunk 1]
```diff
@@ -1,52 +1,146 @@
...
+        """Construct a new InstantDispenser."""
+        super(InstantDispenser, self).__init__()
+
+    def trigger(self):
+        """Dispense the entire portion instantly."""
+        self.driver.dispense_to_scale(777)
+
+
+class BrokenDispenser(_Dispenser):
+    """Dispenses the entire portion at once."""
```
Comment (zzv2) [inline miso_peripherals/src/miso_peripherals/hopper/sim_driver.py:80]: Should this be different than the doc for `InstantDispenser`?

**Review by zzv2 — APPROVED** [review]

File: miso_peripherals/src/miso_peripherals/hopper/sim_driver.py (hunk 1 above)
Comment (luke-powers) [reply miso_peripherals/src/miso_peripherals/hopper/sim_driver.py:80]: :+1:

Comment (rsinnet) [reply miso_peripherals/src/miso_peripherals/hopper/sim_driver.py:80]: Fixed.

**Comment by rsinnet** [comment]
Tested through manual testing.

**Review by luke-powers — APPROVED** [review]

**Pull request merged**